snippets.json

```diff
@@ -0,0 +1,23 @@
+{
+	"insertMarkdownLintDisableComment": {
+		"prefix": "markdownlint-disable",
+		"body": "<!-- markdownlint-disable$1 -->",
+		"description": "Disables one or more markdownlint rules by name (e.g. MD013), alias (e.g. first-heading-h1), or tag (e.g. indentation). Multiple rules are space-delimited (e.g. MD018 MD019). If no rules are specified, then all rules are disabled unconditionally. Takes effect starting with the line this comment is on."
+	},
+	"insertMarkdownLintEnableComment": {
+		"prefix": "markdownlint-enable",
+		"body": "<!-- markdownlint-enable -->",
```

Comment:
Is the $1 needed here too?

Reply:
Yes, thank you. Already fixed that in my most recent amendment as well.